test/PanelSpec.js

```diff
@@ -96,11 +87,13 @@ describe('<Panel>', () => {
 
   it('Should have collapse classes', () => {
     const instance = ReactTestUtils.renderIntoDocument(
-      <Panel collapsible expanded>
-        Panel content
+      <Panel collapsible>
+        <Panel.Body>
+          Panel content
+        </Panel.Body>
       </Panel>
     );
-    assert.ok(ReactDOM.findDOMNode(instance).querySelector('.panel-collapse.collapse.in'));
+    assert.ok(ReactDOM.findDOMNode(instance).querySelector('.panel-collapse.collapse'));
```

Comment:
you can jsut test that `panel-collapse` is present the `Collapse` component tests already handle the rest